drivers/hmis_csv_twenty_twenty/app/views/hmis_csv_twenty_twenty/import_logs/_import_log.haml

```diff
@@ -33,7 +33,7 @@
                       %tr
                         %td= filename
                         %td= number_with_delimiter(v['total_lines'])
-                        %td= number_with_delimiter(v['lines_loaded'])
+                        %td= link_to_if v['lines_loaded']&.positive?, number_with_delimiter(v['lines_loaded']), hmis_csv_twenty_twenty_loaded_path(@import.loader_log, file: filename)
                         %td= link_to_unless v['total_errors'] == 0, number_with_delimiter(v['total_errors']), hmis_csv_twenty_twenty_loader_error_path(@import.loader_log, file: filename)
```

Comment:
I like your use of `link_to_if ... positive?` instead of comparing to zero... -- change these to use that?

Reply:
done.